src/cljs/proton/core.cljs

```diff
@@ -108,9 +109,7 @@
 
           ;; wipe existing config
           (atom-env/insert-process-step! "Wiping existing configuration")
-          (doall (map atom-env/unset-config! (filter #(not
-                                                        (or (= "core.disabledPackages" %)
-                                                            (= "core.themes" %))) (atom-env/get-all-settings))))
+          (doall (map atom-env/unset-config! (filter #(not (= "core.themes" %)) (atom-env/get-all-settings))))
```

Comment:
You were right with this one. This messes with layers that enable/disable packages on init. Need to put this back in.